test/utils_spec.js

```diff
@@ -1411,6 +1425,47 @@ describe("utils", function () {
         api_key: "1234",
       });
     });
+    it("should allow timestamp in sign_request", function() {
```

Comment:
The tests should test exposed functions. Can you please test Upload and verify that it is passing the signature and timestamp?

Reply:
@brian-cloudinary, can you please complete this task?